### PR TITLE
Adding image magnify component

### PR DESCRIPTION
**Pull request by marjald**
Working on a component to perform image magnification - a feature that was suggested by Nike as a nice to have.   Testing wrapping the react-image-magnifiers package.  @jeankhawand  has agreed to look at storybook code.  @ricmars could you please review and see if suitable for UI Gallery?

### CODE REVIEW

**Comment by vijayjangid** [comment]
Could you please add a Docs.mdx file and add details about the use of the component and it's authoring configurations.
For reference, you can check any exsiting field type component [Example: Checkbox Trigger Component](https://github.com/pegasystems/constellation-ui-gallery/blob/master/src/components/Pega_Extensions_CheckboxTrigger/Docs.mdx)

File: src/components/Pega_Extensions_ImageMagnify/demo.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// @ts-nocheck
+import { withKnobs } from '@storybook/addon-knobs';
```
Comment (vijayjangid) [inline src/components/Pega_Extensions_ImageMagnify/demo.stories.tsx:2]: knobs are deprecated, we're using "controls". Please refer other stories files to provide editable story arguments.

**Comment by ricmars** [comment]
Don't bother with this PR - there are couple of other changes to be fixed - I will work on a new PR and merge the changes - I will leverage as much as possible

**Comment by ricmars** [comment]
PR with the new component has been merged https://github.com/pegasystems/constellation-ui-gallery/pull/59/files

I fixed the story and added documentation to the type + added a Unit test. Also fixed the config.json to set defaults

**Pull request closed without merging**